### DISCUSSION TITLE
Road labels rotation

### DISCUSSION THREAD

**Discussion by johanferguson** [SharpMap/SharpMap#221] [General]
Hi There,

Is it possible to custom rotate labels so I can label street names into the direction of the street?

Greetings 

Johan

**Comment by FObermaier** [comment]
This is what SharpMap does with `LineString` labels out of the box:
![PathLabel_SineCurve_HzCenter_VtTop_000](https://user-images.githubusercontent.com/2065346/118232751-525cc080-b491-11eb-8c78-c967e816e227.png)

It is not what you want?

**Reply by johanferguson** [reply]
Aahhhhh.
Thank you so much. I'll check it out.
